src/styles/product.scss

```
@@ -0,0 +1,143 @@
+.container {
+  --columns-count: 1;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--columns-count), 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
```

The readme says 48px vertically and 46 horizontally
```suggestion
  gap: 48px 46px;
```

---

Yes, I used moyo header from the last task